docs/variability.md

```diff
@@ -89,9 +89,36 @@ If your machine has really limited resources or creating a clean environment has
 
 ### Run Lighthouse Multiple Times
 
-When creating your thresholds for failure, either mental or programmatic, use aggregate values like the median, 90th percentile, or even min instead of single tests.
+When creating your thresholds for failure, either mental or programmatic, use aggregate values like the median, 90th percentile, or even min/max instead of single test results.
 
-The median Lighthouse score of 5 runs is twice as stable as 1 run, and tools like [lighthouse-ci](https://github.com/GoogleChrome/lighthouse-ci/) can run Lighthouse multiple times for you automatically. Using the minimum value is also a big improvement over not testing at all and is incredibly simple to implement, just run Lighthouse up to 5 times until it passes!
+The median Lighthouse score of 5 runs is twice as stable as 1 run, and tools like [lighthouse-ci](https://github.com/GoogleChrome/lighthouse-ci/) can run Lighthouse multiple times for you automatically.
+
+```bash
+npx -p lighthouse-ci lhci collect --url https://example.com -n 5
+npx -p lighthouse-ci lhci upload --target filesystem --outputDir ./path/to/dump/reports
+```
+
+You can then process the reports that are output to the filesystem. Read the [Lighthouse CI documentation](https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/configuration.md#outputdir) for more.
+
+If you're running Lighthouse directly via node, you can also use the `computeMedianRun` function to determine the median using a blend of the performance metrics.
```

Comment:
as i mentioned the other day, i think we might wanna steer programmatic users to always using the CLI via child_process for memory reasons.

maybe tweak this example to just read some saved LHRs from disk?

Reply:
Oh I misunderstood then I thought you meant as anti-concurrency example or long-lived processes.

Sure, do you want this to still be a standalone invoking lighthouse example? Or read from say the lighthouse CI manifest output for multiple runs?

Reply:
> Or read from say the lighthouse CI manifest output for multiple runs?

oohhh sexy. yeah that wfm!

Reply:
I did it using child processes instead for now, lmk if you'd prefer the CI example 👍

Reply:
haha oops, crossed in transit. I'll do both :)